elpis/gui/src/components/App.js

```diff
@@ -1,64 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component} from "react";
```

Comment:
In other code, I've seen spaces for import brackets & no spaces everywhere else, although I think that's personal preference as well :)